src/interface.jl

```diff
@@ -2,58 +2,60 @@
 
 
 """
-    isdensitytype(type::Type)
+    hasdensity(d)::Bool
 
-Returns `true` if `type` is compatible with the `DensityInterface` interface,
-otherwise returns `false` by default.
+Returns `true` if `d` is compatible with the `DensityInterface` interface.
+
+`hasdensity(d) == true` implies that `d` is either a density itself or has an
+associated density, e.g. a PDF or a Radon–Nikodym derivative with an implied
```

Comment:
I'm not sure if the distinction is helpful here, I am worried it creates more confusion. Both for readers not familiar with measure theory and readers familiar with it who might wonder why distinguish pdfs at all.

In any case, I think one should avoid abbreviations and do not use the term "implied":
```suggestion
associated density, e.g. a probability density function or a Radon–Nikodym derivative with respect to a
```